src/webots/nodes/utils/WbNodeOperations.hpp

```diff
@@ -62,6 +62,8 @@ class WbNodeOperations : public QObject {
   bool isSkipUpdates() { return mSkipUpdates; }
   bool isFromSupervisor() { return mFromSupervisor; }
 
+  void setFromSupervisor(const bool value);
```

Comment:
It should be private as it is only called from within the `WbNodeOperations` class.

Reply:
Also note that it is useless to use  `const`  when passing the argument by value because the original value is copied and won't be modified.

```suggestion
  void setFromSupervisor(bool value);
```

Same for WbProtoManager::setImportedFromSupervisor`.